artifacts/experimental/etcd.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: core-etcd
+  labels:
+    app: etcd
+spec:
+  containers:
+  - name: etcd
+    image: quay.io/coreos/etcd:v3.2.18
+    env:
+    - name: ETCD_CERT_FILE
+      value: "/etc/server-certs/server.pem"
```

Comment:
this wasn't here before, right? Adding this means you have to add trust certs to have the proxy trust the core etcd, right?